apps/web/src/app/(private)/settings/apikeys/new/page.tsx

```diff
@@ -0,0 +1,16 @@
+'use client'
```

Comment:
Don't use client on pages. Pages are always SSR

Reply:
Okay, I just based the page on the destroy provider one, which also has `use client`: `apps/web/src/app/(private)/settings/apikeys/[providerApiKeyId]/destroy/page.tsx`, maybe it is also wrong?

I see other also doing the same, ex: `apps/web/src/app/(private)/settings/users/[userId]/destroy/page.tsx`...

Reply:
Is not super important. You can ignore this comment. But using `use client` on page makes me feel a bit dirty tbh 😂. Yes, pure OCD